packages/table/src/TableHeaderCell.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import classNames from 'classnames';
 import { ExtendedTableHeader, IdType } from './types/ReactTable';
 import { TableSort } from './types/TableSort';
+import { UncontrolledTooltip } from 'reactstrap';
 
 type Props<T extends IdType> = {
   /** This is a unique id that is prepended to the element **/
   id?: string;
-  /**  **/
+  /** The react-table Column that is build displayed.  **/
```

Comment:
```suggestion
  /** The react-table Column that is build displayed. */
```
---
**QUESTION:** Also I am not certain what this statement means, should it be "that is built and displayed."?

Reply:
I am not getting any lint errors here. Yeah, if you see something you can just comment once and say "I am seeing this throughout" to help make it easier to go through review comments.